ks3-standard/src/commonMain/kotlin/io/ks3/standard/ResilientListSerializer.kt

```diff
@@ -0,0 +1,36 @@
+package io.ks3.standard
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.SerializationStrategy
+import kotlinx.serialization.builtins.ListSerializer
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import kotlinx.serialization.json.JsonDecoder
+import kotlinx.serialization.json.jsonArray
+import kotlinx.serialization.serializer
+
+inline fun <reified T> resilientListSerializer(explicitElementSerializer: KSerializer<T>? = null) =
+   object : KSerializer<List<T>> {
+      private val elementSerializer = explicitElementSerializer ?: serializer<T>()
+      private val serializer: SerializationStrategy<List<T>> = ListSerializer(elementSerializer)
+      override val descriptor: SerialDescriptor = serializer.descriptor
+
+      override fun deserialize(decoder: Decoder): List<T> {
+         if (decoder !is JsonDecoder) error("This deserializer can only be used with Json")
+         val json = decoder.json
+         val list = decoder.decodeJsonElement()
+         return list
+            .jsonArray
+            .mapNotNull {
+               runCatching {
+                  json.decodeFromJsonElement(elementSerializer, it)
+               }.getOrNull()
```

Comment:
```suggestion
               try {
                  Json.decodeFromJsonElement(elementSerializer, it)
               } catch (ex: IllegalArgumentException) {
                  null // ignore failures
               }
```

I think it's better to try and only catch the exception that is documented by `decodeFromJsonElement()`